robot/client/client.go

```diff
@@ -935,3 +941,32 @@ func (rc *RobotClient) StopAll(ctx context.Context, extra map[resource.Name]map[
 	_, err := rc.client.StopAll(ctx, &pb.StopAllRequest{Extra: e})
 	return err
 }
+
+// ModuleLog sends a log entry to the server. To be used by Golang modules wanting to log over gRPC and not
+// by normal Golang SDK clients.
+func (rc *RobotClient) ModuleLog(ctx context.Context, log zapcore.Entry, fields []zapcore.Field) error {
```

Comment:
would change name to `Log` after api

Reply:
While this function does leverage the `Log` endpoint, it _is_  sort of specific to module logging. See the usage of the `moduleLogTSKey` and the leaving out of `Host` because it's not meaningful for modules. If you feel strongly that this should be named `Log`, I might make it more generic by adding a `forModule bool` param or something to attach the `moduleLogTSKey`. Thoughts?

Reply:
I think making it more generic would be ideal, but not through using a `forModule` flag. Can we just add moduleLogTSKey as an extra field?

Reply:
😮 that's a great idea. Done! Let me know if I interpreted that correctly.